src/modules/oal/consumer/oal_consumer.cpp

```diff
@@ -96,18 +122,59 @@ class device
     }
 
     ALCdevice* get() { return device_; }
+
+  private:
+    ALCchar* iterateDevices(const char* list)
+    {
+        ALCchar* result = nullptr;
+        ALCchar* ptr;
+
+        // generate ascii string for comparison purposes vs what openAL provides
+        std::string short_device_name(config.audio_device_name.begin(), config.audio_device_name.end());
```

Comment:
This should use the `u8()` from https://github.com/CasparCG/server/blob/master/src/common/utf.cpp to ensure the value is converted appropriately

Reply:
Ok, I have made some changes based on your feedback, appears to be working.